### PR TITLE
Update everything to support api_post_method

### PR DESCRIPTION
**Pull request by A5rocks**
Fixes https://github.com/botblock/data/issues/45

### CODE REVIEW

File: schema/list.json [hunk 1]
```diff
@@ -123,6 +124,15 @@
             "title": "List API guild count POST endpoint",
```
Comment (MattIPv4) [inline schema/list.json:124]:
```suggestion
            "title": "List API guild count submission endpoint",
```

File: schema/list.json [hunk 2]
```diff
@@ -123,6 +124,15 @@
...
             "description": "The endpoint URL for posting a bot's guild count to the list, if known, with `:id` as placeholder for bot Id."
```
Comment (MattIPv4) [inline schema/list.json:125]:
```suggestion
            "description": "The endpoint URL for submitting a bot's guild count to the list, if known, with `:id` as placeholder for bot Id."
```

File: schema/list.json [hunk 3]
```diff
@@ -123,6 +124,15 @@
...
         },
+        "api_post_method": {
+            "$id": "#/properties/api_post_method",
+            "type": [
+                "string",
+                "null"
+            ],
+            "title": "List API guild count method to use",
```
Comment (MattIPv4) [inline schema/list.json:133]:
```suggestion
            "title": "List API guild count HTTP method",
```

File: schema/list.json [hunk 4]
```diff
@@ -123,6 +124,15 @@
...
+            "description": "The method to use for submitting a bot's guild count to the list via `api_post`."
```
Comment (MattIPv4) [inline schema/list.json:134]:
```suggestion
            "description": "The HTTP method to use for submitting a bot's guild count to the list via `api_post`. Defaults to `POST` if null."
```

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
A few small wording tweaks :)

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Ah! And the OpenAPI schema needs updating too please :) The [examples](https://github.com/botblock/data/blob/acffab0fe6baa8f79ea676f75367f41696f44cd9/openapi.yml#L348) and the [list schema](https://github.com/botblock/data/blob/acffab0fe6baa8f79ea676f75367f41696f44cd9/openapi.yml#L551) will need this new prop

**Comment by MattIPv4** [comment]
@A5rocks Apologies for the delay here -- it looks like you've got one conflict that needs resolving.

**Comment by A5rocks** [comment]
Alright, thanks! Should be fixed now.

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**